maintenance/importConfig.ts

```diff
@@ -308,7 +435,8 @@ async function parseOzwProduct(
 		.basename(product.config, ".xml")
 		.toLocaleUpperCase();
 
-	const productName = product.name;
+	// any products descriptions have productName in it, remove it
+	const productName = product.name.replace(productLabel, "");
```

Comment:
Lets see how this turns out, maybe we need to limit it to the start and end of the string